srcpkgs/apcupsd/template

```diff
@@ -0,0 +1,55 @@
+# Template file for 'apcupsd'
+pkgname=apcupsd
+version=3.14.14
+revision=1
+build_style=gnu-configure
+configure_args="
+ --sysconfdir=/etc/apcupsd
+ --with-pwrfail-dir=/etc/apcupsd
+ --with-upstype=usb --with-upscable=usb --enable-usb --with-dev=
+ --enable-modbus-usb
+ --enable-net --enable-pcnet
+ --enable-snmp
+ --disable-gapcmon
+ --with-nis-port=3551"
+hostmakedepends="autoconf automake"
+makedepends="libusb-compat-devel net-snmp-devel"
+depends="libusb-compat net-snmp nullmailer"
+conf_files="/etc/apcupsd/*"
+short_desc="APC UPS daemon with integrated tcp/ip remote shutdown"
+maintainer="Case_Of <voidlinux@caseof.fr>"
+license="GPL-2.0-only"
+homepage="http://www.apcupsd.org/"
+distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
+checksum="db7748559b6b4c3784f9856561ef6ac6199ef7bd019b3edcd7e0a647bf8f9867"
+
+pre_configure() {
+	export SHUTDOWN=/usr/bin/shutdown
+	export WALL=/usr/bin/wall
+	export APCUPSD_MAIL=/usr/bin/mail
+}
+
+do_install() {
```

Comment:
any reason not to make use of the default `make install`  and do cleanup in post_install if required ?